doc/changelog/08-tools/10947-coq-makefile-dep.rst

```diff
@@ -0,0 +1,5 @@
+- Renamed `VDFILE` from `.coqdeps.d` to `.<CoqMakefile>.d` in the `coq_makefile`
+  utility, where `<CoqMakefile>` is the name of the output file given by the
+  `-o` option, so that all the names of files generated and included by
+  `<CoqMakefile>` are prefixed by the given filename.
```

Comment:
This isn't quite true as the dependencies for `foo.ml` are put in `foo.ml.d`.

Reply:
Do you think it is a misleading communication?

Reply:
I don't know if anyone who cares about the coq_makefile generated filenames would misunderstand.
I guess the main point of this PR is to make sure coqmakefiles for disjoint _coqprojects don't interfere, in which case it works fine but the intention could be clarified in the changelog.
If we also care about non-disjoint _coqprojects it's unclear what we would even want, so we may assume that any `foo.ml.d` belongs to a unique _coqproject and therefore to a unique makefile.

Reply:
@SkySkimmer  Updated: `In this way two generated makefiles can coexist in the same directory.` I didn't explain that data race problem on `foo.ml.d` files but is this fine?